NuRadioMC/SignalProp/analyticraytracing.py

```diff
@@ -901,12 +899,9 @@ def dt(t, C_0, frequency):
                         elif idx == -1:
                             idx = 0
                         
-                        integrand = integrate.quad(self.ds, segments[idx], segments[idx + 1], args=(C_0), epsrel=1e-2, points=[z_turn])[0]
-                        attenuation = (attenuation_util.get_attenuation_length(z_turn, f, self.attenuation_model) for f in freqs)
-                        att_int = np.array(
-                            [integrand, attenuation])
-
-                        attenuation_exp_tmp[:, idx] = att_int
+                        
+                        att = np.array([attenuation_util.get_attenuation_length(z_turn, f, self.attenuation_model) for f in freqs])
+                        attenuation_exp_tmp[:, idx] = 1. / att
```

Comment:
I am a bit confused. I have a PR #908 which fixes a bug which this PR apparently also fixes but the results looks different. @christophwelling why don't you do `integrand / att`? Is `integrand` by design 1?

Reply:
Okay I convinced myself with the example in NuRadio that this is wrong and the correct equation is `integrand / att`. Will change it.